dts/bindings/dma/espressif,esp32-gdma.yaml

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2022 Espressif Systems (Shanghai) Co., Ltd.
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  Espressif's GDMA (General Direct Memory Access) Node
+
+  General Direct Memory Access (GDMA) is a feature that allows
+  peripheral-to-memory, memory-to-peripheral, and memory-to-memory
+  data transfer at a high speed.
+
+  The GDMA controller in ESP32-C3 has six independent channels,
```

Comment:
Is this IP block only used on the C3? Is it on the new P4 as well? If so this description might be worth updating to be more generic describing the IP itself and its usage rather than explicitly describing which peripherals on a particular part can be used with it.

Reply:
Indeed the GDMA is present on other SoCs, with some small differences between them. But for the time being it is only supported on C3. I think that for the sake of documentation and reference it is better to explicit describe those that are currently supported. As soon as we add support for other SoC, we add them to the binding documenting its particularity.